app/assets/stylesheets/modern-business.scss

```diff
@@ -150,6 +150,14 @@ header.carousel .fill#slider-3 {
   font-size: 20px;
 }
 
+#index-names {
+  color: #A30505;
```

Comment:
Color literals like `#A30505` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#A30505` should be written as `#a30505`